src/OpenTelemetry.Exporter.Console/OpenTelemetryBuilderExtensions.cs

```diff
@@ -21,62 +21,28 @@ namespace OpenTelemetry.Exporter.Console
-        /// Adds new processing pipeline and registers a ConsoleActivity exporter to it.
-        /// </summary>
-        /// <param name="builder">Open Telemetry builder to use.</param>
-        /// <param name="configure">Exporter configuration options.</param>
-        /// <returns>The instance of <see cref="OpenTelemetryBuilder"/> to chain the calls.</returns>
-        public static OpenTelemetryBuilder UseConsoleActivityExporter(this OpenTelemetryBuilder builder, Action<ConsoleActivityExporterOptions> configure)
-        {
-            if (builder == null)
-            {
-                throw new ArgumentNullException(nameof(builder));
-            }
-
-            if (configure == null)
-            {
-                throw new ArgumentNullException(nameof(configure));
-            }
-
-            var exporterOptions = new ConsoleActivityExporterOptions();
-            configure(exporterOptions);
-            var consoleExporter = new ConsoleActivityExporter(exporterOptions);
-            return builder.AddProcessorPipeline(pipeline => pipeline.SetExporter(consoleExporter));
-        }
-
         /// <summary>
         /// Registers a ConsoleActivity exporter to a processing pipeline.
         /// </summary>
         /// <param name="builder"><see cref="OpenTelemetryBuilder"/> builder to use.</param>
         /// <param name="configure">Exporter configuration options.</param>
         /// <param name="processorConfigure">Activity processor configuration.</param>
         /// <returns>The instance of <see cref="OpenTelemetryBuilder"/> to chain the calls.</returns>
-        public static OpenTelemetryBuilder UseConsoleActivityExporter(this OpenTelemetryBuilder builder, Action<ConsoleActivityExporterOptions> configure, Action<ActivityProcessorPipelineBuilder> processorConfigure)
+        public static OpenTelemetryBuilder UseConsoleActivityExporter(this OpenTelemetryBuilder builder, Action<ConsoleActivityExporterOptions> configure = null, Action<ActivityProcessorPipelineBuilder> processorConfigure = null)
         {
             if (builder == null)
             {
                 throw new ArgumentNullException(nameof(builder));
             }
 
-            if (configure == null)
-            {
-                throw new ArgumentNullException(nameof(configure));
-            }
-
-            if (processorConfigure == null)
-            {
-                throw new ArgumentNullException(nameof(processorConfigure));
-            }
-
             return builder.AddProcessorPipeline(pipeline =>
             {
                 var exporterOptions = new ConsoleActivityExporterOptions();
-                configure(exporterOptions);
+                configure?.Invoke(exporterOptions);
 
                 var consoleExporter = new ConsoleActivityExporter(exporterOptions);
                 pipeline.SetExporter(consoleExporter);
-                processorConfigure(pipeline);
+                processorConfigure?.Invoke(pipeline);
```

Comment:
its possible for user to set a different Exporter using processorConfigure action. Since by definition of this method `UseConsoleActivityExporter`, we expect to end up with ConsoleExporter, should we do this before setting the Exporter to console? So that even if user set exporter to something, it'll be overrritten to consoleexporter?

Reply:
Yes. I will modify swap the lines of code. Before sending PR, I had it in this way and changed it as TraceBuilderExtensions registers span custom processor at the end.

Reply:
Fixed.

Reply:
Thanks. Lets revisit this area to see how to make this even better. Right now, if user set own Exporter, we just override it. 
We need to figure out a better option.